javascript/test/aspects-tests.js

```diff
@@ -359,5 +359,83 @@ describe('Aspects Tests', function () {
             assert.strictEqual(minesweeperProcess.stdout.toString(), expectedOutput.join(''), "Wrong output")
             // instead of checking one by one the boars, we assert over the entire output
         });
+
+        it('Win Minesweeper after 3 steps where one increments both mines and moves', function () {
+            // Creates the simple.cfg file
+            const lines = ['..*', '...', '...'];
+            fs.writeFileSync(tempDir + "simple.cfg", lines.join('\n') + '\n')
+
+            var expectedOutput = [
+                "╔═══════════╗\n",
+                "║Mines: 1   ║\n",
+                "║Moves: 0   ║\n",
+                "╚═══════════╝\n",
+                "┌───┬───┬───┐\n",
+                "│   │   │   │\n",
+                "├───┼───┼───┤\n",
+                "│   │   │   │\n",
+                "├───┼───┼───┤\n",
+                "│   │   │   │\n",
+                "└───┴───┴───┘\n",
+                "╔═══════════╗\n",
+                "║           ║\n",
+                "╚═══════════╝\n",
+                ">",
+                "╔═══════════╗\n",
+                "║Mines: 0   ║\n",
```

Comment:
Why would the number of remaining mines decrease, if there was a flag move on a field that did not contain a mine? The first move is "2 2 F" and the only mine on the board is on 1 3.

Reply:
never mind. I apparently misunderstood the task. Thanks for the tests otherwise I would not have noticed :+1: